build.sbt

```diff
@@ -44,7 +44,9 @@ lazy val scoverageSettings: Seq[Setting[_]] = Seq(
     "features\\..*",
     "test\\..*",
     ".*(BuildInfo|Routes|Options).*",
-    "logger.*\\(.*\\)"
+    "logger.*\\(.*\\)",
+    ".*views.html.components\\..*",
+    ".*views.html.partials\\..*"
```

Comment:
Why do we need to exclude these?  The components and partials should be used in a view somewhere

Reply:
I felt like they were negatively feeding into lower coverage numbers without us having specific tests for them and I didn't want to introduce those; I prefer the idea of them being tested via the view tests of the views that use them.

Reply:
@SteveSugden , I have reverted these changes.